plugins/c8y_log_plugin/src/main.rs

```diff
@@ -108,14 +108,19 @@ async fn run(
                             "522" => {
                                 info!("Log request received: {payload}");
                                 // retrieve smartrest object from payload
-                                let smartrest_obj = SmartRestLogRequest::from_smartrest(payload)?;
-                                handle_logfile_request_operation(
-                                    &smartrest_obj,
-                                    &plugin_config,
-                                    mqtt_client,
-                                    http_client,
-                                )
-                                .await
+                                let smartrest_obj = SmartRestLogRequest::from_smartrest(payload);
+                                if let Ok(smartrest_obj) = smartrest_obj {
```

Comment:
I prefer to rename Result value.
```suggestion
                                let maybe_smartrest_obj = SmartRestLogRequest::from_smartrest(payload);
                                if let Ok(smartrest_obj) = maybe_smartrest_obj {
```